src/main/java/com/powsybl/openloadflow/network/LfArea.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2024, Coreso SA (https://www.coreso.eu/) and TSCNET Services GmbH (https://www.tscnet.eu/)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.openloadflow.network;
+
+import com.powsybl.openloadflow.util.Evaluable;
+
+import java.util.Map;
+import java.util.Set;
+import java.util.function.Supplier;
+
+/**
+ * @author Valentin Mouradian {@literal <valentin.mouradian at artelys.com>}
+ */
+public interface LfArea extends PropertyBag {
+    String getId();
+
+    double getInterchangeTarget();
```

Comment:
please prepare also a setter

Reply:
added in c2f88b5fdd845f63af4e002a2722d160cbcf8a3b